product_docs/docs/pgd/4/deployments/installing_cli.mdx

```diff
@@ -1,27 +1,17 @@
 ---
-title: "Installing the EDB Postgres Distributed Command Line Interface"
-navTitle: "Installing the PGD CLI"
+title: "Installing a standalone instance of PGD CLI"
+navTitle: "Installing PGD CLI"
 ---
 
 
-## Installing with TPAexec
```

Comment:
This is necessary. PGD CLI is totally different than PGD including everything packaging, configuration etc.